src/main/java/controller/CalculatorApp.java

```diff
@@ -0,0 +1,17 @@
+package controller;
+
+import domain.Calculator;
+import domain.Tokens;
+import view.InputView;
+import view.ResultView;
+
+public class CalculatorApp {
+    public static void main(String[] args) {
+        String expression = InputView.getExpression();
+        Tokens tokens = new Tokens(expression);
+        tokens.validate();
```

Comment:
```suggestion
        Tokens tokens = new Tokens(expression);
```
생성자 내부에서 유효성검사까지 포함시킬 수 있어보여요

Reply:
감사합니다. 반영했습니다.